packages/migrate/src/types.ts

```diff
@@ -137,6 +137,197 @@ export namespace EngineArgs {
+      | AffectedEnum[]
+      | AffectedEnumAndValue[]
+      | null
+  }
+
+  interface IntrospectionWarningsUnhandled extends IntrospectionWarning {
+    code: -1 // -1 doesn't exist, it's just for the types
+    affected: any
+  }
+  interface IntrospectionWarningsInvalidReintro extends IntrospectionWarning {
+    code: 0
+    affected: null
+  }
+  interface IntrospectionWarningsMissingUnique extends IntrospectionWarning {
+    code: 1
+    affected: AffectedModel[]
+  }
+  interface IntrospectionWarningsEmptyFieldName extends IntrospectionWarning {
+    code: 2
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsUnsupportedType extends IntrospectionWarning {
+    code: 3
+    affected: AffectedModelAndFieldAndType[]
+  }
+  interface IntrospectionWarningsInvalidEnumName extends IntrospectionWarning {
+    code: 4
+    affected: AffectedEnumAndValue[]
+  }
+  interface IntrospectionWarningsCuidPrisma1 extends IntrospectionWarning {
+    code: 5
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsUuidPrisma1 extends IntrospectionWarning {
+    code: 6
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsFieldModelReintro extends IntrospectionWarning {
+    code: 7
+    affected: AffectedModel[]
+  }
+  interface IntrospectionWarningsFieldMapReintro extends IntrospectionWarning {
+    code: 8
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsEnumMapReintro extends IntrospectionWarning {
+    code: 9
+    affected: AffectedEnum[]
+  }
+  interface IntrospectionWarningsEnumValueMapReintro extends IntrospectionWarning {
+    code: 10
+    affected: AffectedEnum[]
+  }
+  interface IntrospectionWarningsCuidReintro extends IntrospectionWarning {
+    code: 11
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsUuidReintro extends IntrospectionWarning {
+    code: 12
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsUpdatedAtReintro extends IntrospectionWarning {
+    code: 13
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsWithoutColumns extends IntrospectionWarning {
+    code: 14
+    affected: AffectedModel[]
+  }
+  interface IntrospectionWarningsModelsWithIgnoreReintro extends IntrospectionWarning {
+    code: 15
+    affected: AffectedModel[]
+  }
+  interface IntrospectionWarningsFieldsWithIgnoreReintro extends IntrospectionWarning {
+    code: 16
+    affected: AffectedModelAndField[]
+  }
+  interface IntrospectionWarningsCustomIndexNameReintro extends IntrospectionWarning {
+    code: 17
+    affected: AffectedModelAndIndex[]
+  }
+  interface IntrospectionWarningsCustomPrimaryKeyNamesReintro extends IntrospectionWarning {
+    code: 18
+    affected: AffectedModel[]
+  }
+  interface IntrospectionWarningsRelationsReintro extends IntrospectionWarning {
+    code: 19
+    affected: AffectedModel[]
+  }
+
```

Comment:
It's missing
```
interface IntrospectionWarningsTopLevelItemNameIsADupe extends IntrospectionWarning {
  code: 20
  affected: AffectedTopLevel[]
}
```

Can you add the recent changes from this commit?
https://github.com/prisma/prisma/commit/c2d32eea77e963778ba0b98cc3557ad0de62a5b7#diff-a3ca29a928d7e7c324513f5239c93d68b07cbd190c678587efc3eb2a93c32d82

Reply:
Note: It's the only change since this PR was opened

Reply:
Done